### PR TITLE
retrieve requisition by id

### PR DESCRIPTION
**Pull request by Oluwaseun011**

### CODE REVIEW

File: src/Application/Queries/GetRequisition.cs
Comment (honourableola) [inline src/Application/Queries/GetRequisition.cs]: I think based on the UI/UX frames we'll probably need the name of the submitter (not sure SubmitterId would be needed).

Comment (honourableola) [inline src/Application/Queries/GetRequisition.cs]: I don't think we need the whole Approval flow object as it is in the entity, we might need more specific information which is not directly available in the object. For example, the names of the individuals that has approved (as opposed to Id), the current approver that the request is with... Can you check the new UI/UX details screen to confirm what exactly we need?

**Review by NurudeenAdeyemi — APPROVED** [review]

**Pull request merged**
